examples/blog/fakerest-init.js

```diff
@@ -9,6 +9,35 @@
     restServer.init(apiData);
     restServer.toggleLogging(); // logging is off by default, enable it
 
+    restServer.addRequestInterceptor(function(request) {
+        var filters = request.params.filter;
+        request.params.filter = function(item) {
+            for (var key in filters) {
+                if ('search' === key) {
```

Comment:
I've updated fakerest to implement that on the API side (refs https://github.com/marmelab/FakeRest/pull/10)